pocket_coffea/workflows/base.py

```diff
@@ -767,6 +773,7 @@ def postprocess(self, accumulator):
                         continue
                     h *= scale_genweight[sample]
             accumulator["scale_genweight"] = scale_genweight
+            """
```

Comment:
At the moment this part is just commented it out but could be removed.

Reply:
No dangling comment code..